cms/migrations/0015_auto_20160914_2033.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-09-14 20:33
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import wagtail.wagtailcore.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0014_html_in_faq_answers'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='programpage',
+            name='background_image',
+            field=models.ForeignKey(blank=True, help_text='The hero image on the program page', null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='wagtailimages.Image'),
+        ),
+        migrations.AlterField(
+            model_name='programpage',
+            name='description',
+            field=wagtail.wagtailcore.fields.RichTextField(blank=True, help_text='The description shown on the program page'),
+        ),
+        migrations.AlterField(
+            model_name='programpage',
+            name='external_program_page_url',
+            field=models.URLField(blank=True, help_text='If this field is set the program page link on the home page will go to this URL.', null=True),
+        ),
+        migrations.AlterField(
+            model_name='programpage',
+            name='program',
+            field=models.OneToOneField(help_text='The program for this page', null=True, on_delete=django.db.models.deletion.SET_NULL, to='courses.Program'),
+        ),
+        migrations.AlterField(
+            model_name='programpage',
+            name='thumbnail_image',
+            field=models.ForeignKey(blank=True, help_text='Thumbnail size must be at least 690x530 pixels. Thumbnails are cropped down to this size, preserving aspect ratio.', null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='wagtailimages.Image'),
+        ),
+    ]
```

Comment:
Hmm, looks like you're right.  This was probably created accidentally when we made migrations and we didn't notice.  I'll remove it.